sauce-connect.rb

```diff
@@ -1,7 +1,7 @@
 require "formula"
 class SauceConnect < Formula
   homepage "https://docs.saucelabs.com/reference/sauce-connect/"
-  url "https://saucelabs.com/downloads/sc-4.3.8-osx.zip"
+  url "https://saucelabs.com/downloads/sc-4.3.9-osx.zip"
   sha1 "9d9ecb49fbea70186d08aac7de6032546d856379"
```

Comment:
I'm not entirely familiar with this homebrew syntax, but wouldn't this hash need to reflect the new sc package?